### PR TITLE
feat/140 - Multi-chain configurations

### PR DESCRIPTION
**Pull request by jurevans**
See #140 

This PR is to add multiple chain definitions shared by both the extension and namada-interface.

- [x] Create a chain type definition to cover all requirements for the interface (replacing usage of Keplr's `ChainInfo` type)
- [x] Import all chain configs on `namada-interface`, and import Namada chain config into `extension`
- [x] Allow overrides specific to environment
- [x] Configs should identify which _extension_ is required for connecting accounts (e.g., `anoma`, `keplr`, `metamask`, etc.)

__NOTE__ See  `apps/namada-interface/.env.sample` for update environment config values (I've added variables for Cosmos & Osmosis as well) - We can use defaults, but for local testing, you may need to update these. These `.env` values are also imported into the `webpack.config.js` build which makes them available in the extension. They aren't required, but will likely need to be updated for local testing.

__NOTE__ The Keplr integration is currently a bit wonky, but that is to be addressed in a separate issue (see #18 and #5).

## New variable overrides for environments

All of the following defaults can be overridden via the `.env` file in `apps/namada-interface` - these also apply to the extension, which loads the same `.env` (we only need one `.env` with the extension's `webpack.config.js` pointing to this one)

```bash
# NAMADA
REACT_APP_NAMADA_ALIAS=Namada Testnet
REACT_APP_NAMADA_CHAIN_ID=namada-1.5.32ccad5356012a7
REACT_APP_NAMADA_URL=http://127.0.0.1:26657
REACT_APP_NAMADA_BECH32_PREFIX=atest

# COSMOS
REACT_APP_COSMOS_ALIAS=Cosmos Testnet
REACT_APP_COSMOS_CHAIN_ID=cosmos-testnet.123412341234
REACT_APP_COSMOS_URL=http://127.0.0.1:12345

# OSMOSIS
REACT_APP_OSMOSIS_ALIAS=Osmosis Testnet
REACT_APP_OSMOSIS_CHAIN_ID=osmosis-testnet.123412341234
REACT_APP_OSMOSIS_URL=http://127.0.0.1:54321
```

### CODE REVIEW

File: packages/rpc/src/SocketClient.ts [hunk 1]
```diff
@@ -25,7 +25,11 @@ class SocketClient extends RpcClientBase {
   private _client: WebsocketClient | null = null;
 
   public connect(): void {
-    this._client = new WebsocketClient(this.endpoint, (e: Error) => {
```
Comment (jurevans) [inline packages/rpc/src/SocketClient.ts:28]: NOTE: This file is going away completely, so I am just parsing the RPC HTTP URL here to force it into the appropriate websocket URL, but we won't be using websockets soon, this is just temporary.

File: packages/chains/src/chains/cosmos.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import { Chain, Extensions } from "@anoma/types";
+
+const DEFAULT_ALIAS = "Cosmos Testnet";
+const DEFAULT_CHAIN_ID = "cosmos-75a7e12.69483d59a9fb174";
```
Comment (memasdeligeorgakis) [inline packages/chains/src/chains/cosmos.ts:4]: Do we have any way to override these conveniently?

Comment (memasdeligeorgakis) [reply packages/chains/src/chains/cosmos.ts:4]: But likely that is not MVP

**Review by memasdeligeorgakis — APPROVED** [review]

File: packages/chains/src/chains/cosmos.ts (hunk 2 above)
Comment (jurevans) [reply packages/chains/src/chains/cosmos.ts:4]: We can provide overrides in the `.env` file, which needs to be there when we want to point to a local testnet (for example) - these get loaded from `process.env` just below:

**Pull request merged**